config/crd/bases/keyserver.kairos.io_sealedvolumes.yaml

```diff
@@ -37,15 +37,32 @@ spec:
             properties:
               TPMHash:
                 type: string
-              partitionSecrets:
-                additionalProperties:
+              partitions:
+                items:
+                  description: 'PartitionSpec defines a Partition. A partition can
+                    be identified using any of the fields: Label, DeviceName, UUID.
+                    The Secret defines the secret which decrypts the partition.'
                   properties:
-                    name:
+                    deviceName:
                       type: string
-                    path:
+                    label:
                       type: string
+                    secret:
+                      properties:
+                        name:
+                          type: string
+                        path:
+                          type: string
+                      type: object
+                    uuid:
+                      type: string
+                  required:
+                  - deviceName
```

Comment:
this doesn't look right, are all of them required? miss `omitempty` in the spec struct probably

Reply:
correct, added now and re-generated the manifests